sdk/azcore/runtime/policy_logging.go

```diff
@@ -30,7 +33,49 @@ func NewLogPolicy(o *policy.LogOptions) policy.Policy {
 	if o == nil {
 		o = &policy.LogOptions{}
 	}
-	return &logPolicy{options: *o}
+	// construct default hash set of allowed headers
+	allowedHeaders := map[string]struct{}{
```

Comment:
This set was taken from the [.NET](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/DiagnosticsOptions.cs#L21) implementation.